content/geometry/PointInsideHull.h

```diff
@@ -1,13 +1,13 @@
 /**
- * Author: Johan Sannemo
- * Date: 2017-04-13
+ * Author: chilli
+ * Date: 2019-05-17
  * License: CC0
- * Source: Inspired by old, broken tinyKACTL
- * Description: Determine whether a point t lies inside a given polygon (counter-clockwise order).
- * The polygon must be such that every point on the circumference is visible from the first point in the vector.
- * It returns 0 for points outside, 1 for points on the circumference, and 2 for points inside.
+ * Source: https://github.com/ngthanhtrung23/ACM_Notebook_new
+ * Description: Determine whether a point t lies inside a convex hull. Returns
```

Comment:
adding "(counter-clockwise order, with no colinear points)" would be useful, if that's the case

Reply:
can also shorten the strictness comment slightly into "if strict is true, points on the boundary are not included" or some such